core/controllers/topic_viewer.py

```diff
@@ -106,6 +107,13 @@ def get(self, topic_name):
             for skill_id in assigned_skill_ids:
                 degrees_of_mastery[skill_id] = None
 
+        show_train_tab = False
```

Comment:
train_tab_should_be_displayed

(sorry, I know it's longer, but this follows the convention of starting with a verb only for methods/functions).

Reply:
Done.